### PR TITLE
Ectyper

### PR DESCRIPTION
**Pull request by UBCSamSung**
* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/ectyper/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    summary: Ectyper is a python script for serotyping e.coli genomes
+    author: Camille La Rose, Chad Laing, Sam Sung
+    home: https://github.com/phac-nml/ecoli_serotyping
+
+package:
+    name: ectyper
+    version: 0.0.8
+
+source:
+    url: https://github.com/phac-nml/ecoli_serotyping/archive/0dbfb220d76f7c0fe840ed7c304d8cc1f6e994db.zip
```
Comment (bgruening) [inline recipes/ectyper/meta.yaml]: There is a 0.1 alpha release: https://github.com/phac-nml/ecoli_serotyping/releases
Could you maybe ask for a new release tarball?

**Comment by UBCSamSung** [comment]
@bgruening Thank you for reviewing. The conda package now use latest release tarball.

File: recipes/ectyper/meta.yaml (hunk 1 above)
Comment (UBCSamSung) [reply recipes/ectyper/meta.yaml]: It is done.

File: recipes/ectyper/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    name: ectyper
+    version: 0.0.8
+
+source:
+    url: https://github.com/phac-nml/ecoli_serotyping/archive/v0.1.tar.gz
+    md5: 06b5793ed0f7000a05e20600b1ac5d0c
+
+requirements:
+    build:
+        - python 3.6.3.*
```
Comment (bgruening) [inline recipes/ectyper/meta.yaml]: Python should not be pinned this way.
Is there any reason to skip python 3.5? Please skip all python version in the build/skip section.

File: recipes/ectyper/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    run:
+        - python 3.6.3.*
+        - pandas 0.21.0.*
```
Comment (bgruening) [inline recipes/ectyper/meta.yaml]: All other pins are here very strict, is this really needed? Biopython for example is stable afaik and does not need to be pinned to 1.69.

**Comment by UBCSamSung** [comment]
@bgruening Thank you for reviewing. I have removed the strict dependency requirement and skipped python 2.7. All checks have passed.

File: recipes/ectyper/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+about:
+    license: MIT
+    summary: Ectyper is a python script for serotyping e.coli genomes
+    author: Camille La Rose, Chad Laing, Sam Sung
+    home: https://github.com/phac-nml/ecoli_serotyping
+
+package:
+    name: ectyper
+    version: 0.1.0
```
Comment (bgruening) [inline recipes/ectyper/meta.yaml]: Now the number is still not the number from the github release. It should be 0.1 isn't it?
You could also use template vars for the version number, this way an upgrade is easier later on.

**Comment by bgruening** [comment]
Thanks @UBCSamSung. Just one last comment.

**Comment by UBCSamSung** [comment]
@bgruening It should be good now.

**Comment by bgruening** [comment]
Thanks @UBCSamSung!

**Pull request merged**